$MirOS: ports/www/dillo/stable/patches/patch-src_IO_about_c,v 1.1 2005/06/19 17:20:19 bsiegert Exp $
--- src/IO/about.c.orig	Wed Apr 26 16:17:25 2006
+++ src/IO/about.c	Mon May 15 17:49:33 2006
@@ -341,7 +341,7 @@ static char *Splash=
 "   <a href='http://www.dillo.org/dillorc'>dillorc</a>\n"
 "   (readable  config)  file within the tarball; It is well commented\n"
 "   and  has  plenty  of  options to customize dillo, so <STRONG>copy\n"
-"   it</STRONG>  to  your  <STRONG>~/.dillo/</STRONG>  directory, and\n"
+"   it</STRONG>  to  your  <STRONG>~/.etc/dillo/</STRONG>  directory, and\n"
 "   modify to your taste.\n"
 " <li> There's documentation for developers in the <CODE>/doc</CODE>\n"
 "   dir  within  the  tarball;  you can find directions on everything\n"
